public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionCompleteProgressBar.js

```diff
@@ -71,6 +71,11 @@ function ModalMissionCompleteProgressBar (uiModalMissionComplete) {
      * @private
      */
     this.update = function (missionDistanceRate, userAuditedDistanceRate, otherAuditedDistanceRate) {
+        // rounding rates to .1 accuracy to avoid floating value arithmetic error
```

Comment:
As per [the style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide), could you change the comment to start with a capital letter and end with a period?